crates/api/src/lib.rs

```diff
@@ -139,6 +140,12 @@ pub async fn match_websocket_operation(
       do_websocket_operation::<TransferCommunity>(context, id, op, data).await
     }
     UserOperation::LeaveAdmin => do_websocket_operation::<LeaveAdmin>(context, id, op, data).await,
+    UserOperation::GetTaglines => {
+      do_websocket_operation::<ListTaglines>(context, id, op, data).await
+    }
+    UserOperation::GetRandomTagline => {
+      do_websocket_operation::<GetRandomTagline>(context, id, op, data).await
+    }
```

Comment:
The GetRandomTagline endpoint is probably unecessary. 

- The random tagline should come back with `GetSiteResponse`. Then you don't have to fetch this every time. 
- The ListTaglines *could* be its own endpoint, or it could also come back with `GetSiteResponse`. I spose its own endpoint is a little better tho.

Reply:
Also you seem to be missing all the other endpoints you have for the HTTP one.

Reply:
Hm okay. Having it in Site was my original thought, but I think the Site object is only refreshed when the lemmy app is loaded, so you would only see one tagline as you navigate around. I was thinking it would be refreshed anytime the Home page is loaded instead. 

Just confirm if that is how you want it, and I will change it, thanks!

Reply:
Hm okay. Having it in Site was my original thought, but I think the Site object is only refreshed when the lemmy app is loaded, so you would only see one tagline as you navigate around. I was thinking it would be refreshed anytime the Home page is loaded instead.

Just confirm if that is how you want it, and I will change it, thanks!

Reply:
Yeah I think so, I don't see why every time you navigate to the home page, you need to see a new tagline. Seems unecessary to do http fetches just for that field.

The alternative would be to include all the taglines in GetSiteResponse, and let the front ends pick a random one whenever they see fit. I'd also be fine with that.

Reply:
Actually, i like that idea of including all taglines. Ill implement it that way, thanks!